### PR TITLE
[main] tcpdump: Update to 4.99.1

### PR DESCRIPTION
**Pull request by rlmenge**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Update to 4.99.1

###### Change Log  

- Update to 4.99.1

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- local build (with RUN_CHECK=y)

### CODE REVIEW

**Review by mfrw — APPROVED** [review]
LGTM

**Review by PawelWMS — APPROVED** [review]
Just a small request to use a symboli link instead of a binary duplicate. Otherwise looks good.

File: SPECS/tcpdump/tcpdump.spec [hunk 1]
```diff
@@ -25,7 +24,17 @@ transmitted or received over a network to which the computer is attached.
...
-make DESTDIR=%{buildroot} install
+# make install installs to /usr/bin in 4.99.1
+# so specify install to sbin instead
+mkdir -p %{buildroot}/%{_sbindir}
+mkdir -p %{buildroot}/%{_mandir}/man1
+
+chmod 755 %{buildroot}/%{_sbindir}
+
+install -m755 tcpdump %{buildroot}%{_sbindir}
+install -m755 tcpdump %{buildroot}%{_sbindir}/tcpdump.%{version}
```
[the remark this answers is not in the document]
Comment (PawelWMS) [reply SPECS/tcpdump/tcpdump.spec:35]:
```suggestion
ln -s tcpdump %{buildroot}%{_sbindir}/tcpdump.%{version}
```

[the remark this answers is not in the document]
Comment (rlmenge) [reply SPECS/tcpdump/tcpdump.spec:36]: Currently, we support two binaries. This is how the `make install` command from upstream operates https://github.com/the-tcpdump-group/tcpdump/blob/master/Makefile.in#L433 . Happy to change though.

**Review by PawelWMS — APPROVED** [review]

File: SPECS/tcpdump/tcpdump.spec (hunk 1 above)
[the remark this answers is not in the document]
Comment (rlmenge) [reply SPECS/tcpdump/tcpdump.spec]: Added symlink to save space. Marking as resolved :)

**Pull request merged**